<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>891</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>수강하는 강좌 목록</string>
   </property>
   <widget class="QTableWidget" name="lect_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>871</width>
      <height>331</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>145</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>과목번호</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>교과목명</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>담당교수</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>강의시간</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>개설학과</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QPushButton" name="checkAttndButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(206, 229, 206);
color:black;
border-color:(0, 0, 0);
border-style:solid;
border-width:1px;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>출결조회</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
